### ISSUE TITLE
Split the code into multiple files

### ISSUE THREAD

**Issue by aappaapp** [voylin/TeleDot#38]
So now, if I understand correctly, all the code except for version checking is inside the `controller.gd` file. This makes the pull requests have conflicts and harder to find the code I am looking for. Therefore, I think we should split the code in `controller.gd` into 2 - 3 files.  (like a `settings.gd` autoload to manage settings)

P.S. We should not split it into too many files since it would break the point of it being simple.

**Comment by voylin** [comment]
I agree that the code should be split. The main intention of splitting it is to make it more clean and not to make it more complex. Having all code for everything within 1 file makes it more complicated even with the comments which are being used to divide the different parts.
